src/Cache/Cache.php

```diff
@@ -168,7 +168,18 @@ protected static function _buildEngine($name)
         }
 
         $config = static::$_config[$name];
-        $registry->load($name, $config);
+
+        try {
+            $registry->load($name, $config);
+        } catch (RuntimeException $e) {
+            if (!array_key_exists('fallback', $config)) {
```

Comment:
I don't think it would log anymore than a normal exception due to a failed init would log, since building only happens once per request and the "bad" object is replaced in the registry.